### DISCUSSION TITLE
Introducing Global Bindings

### DISCUSSION THREAD

**Discussion by jamie-budibase** [Budibase/budibase#12926] [Announcements]
The introduction of Global Bindings means you can now connect any data source to any component on the same screen, eliminating the need for complex nested component structures. 

With this update, the components underlying bindings are built exactly as before, but global resolution means that they can be leveraged anywhere. This update is designed to integrate seamlessly with your existing setup. 

Upgrade to the latest version of Budibase to try Global Bindings today.

[Read the launch blog](https://budibase.com/blog/updates/2024/global-bindings/)

**Comment by josh99smith** [comment]
Great feature! But the new pricing model has priced our non profit out of any of the new updates and features.
